src/qrCard/story.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { text, withKnobs } from '@storybook/addon-knobs'
+import Section from 'layout/section/baseSection'
+import QrCard from './index'
+
+const stories = storiesOf('Widgets|QrCard', module)
+
+stories.addDecorator(withKnobs)
+
+const style: React.CSSProperties = {
+  width: '327px',
```

Comment:
Out of curiosity why 327px? :D

Reply:
just to have a visible story (same size than the sketch design) :)

Reply:
Having this in the storybook might give the impression that it's ok to style the section. I would just drop it completely.